index.bs

```diff
@@ -177,13 +177,10 @@ This specification relies on several other underlying specifications.
 :: The <dfn dictionary>AlgorithmIdentifier</dfn> type and the method for <a>normalizing an algorithm</a> are defined in
     [[WebCryptoAPI#algorithm-dictionary]].
 
-
-
 : Base64url encoding
 :: The term <dfn>Base64url Encoding</dfn> refers to the base64 encoding using the URL- and filename-safe character set defined
     in Section 5 of [[!RFC4648]], with all trailing '=' characters omitted (as permitted by Section 3.2) and without the
-    inclusion of any line breaks, whitespace, or other additional characters. This is the same encoding as used by JSON Web
-    Signature (JWS) [[RFC7515]].
+    inclusion of any line breaks, whitespace, or other additional characters.
```

Comment:
It just seems random. The only places we mention base64url are related to encoding a challenge and a token binding ID in ClientData. Neither of those is in any way related to RFC 7515 at first sight. So why did we pick this RFC to call out as opposed to any other standard that uses base64url?